newsletter august 24 (#800)
RSE-Sheffield/RSE-Sheffield.github.io: _newsletters/2024-08-13-newsletter.md

---
layout: post
title: August 2024 Newsletter
editor: Romain Thomas
slug: 2024-08-13-newsletter

> date: 2024-07-13T17:00:00Z+0100

Reviewer: ```suggestion
date: 2024-08-13T17:00:00Z+0100
```

tags:
category:
link:
description:
type: text
---

## The University of Sheffield Research Software Engineering Community Newsletter August 2024

Welcome to the August 2024 newsletter for the research software community at The University of Sheffield, featuring
news, opportunities, events and training for you.

### News

> Thank you to everyone who applied for the [2024 RSE collaboration call for proposals](https://rse.shef.ac.uk/collaboration/RSEtime_call2024/). This call is now closed. We had 26 applications! We have reviewed all proposals and sent notifications to all applicants.

Reviewer 2: Would we be happy announcing the projects that were awarded support? If so...

```suggestion
Thank you to everyone who applied for the [2024 RSE collaboration call for proposals](https://rse.shef.ac.uk/collaboration/RSEtime_call2024/). This call is now closed. 

We received 26(!) applications from across the University. We have reviewed all proposals and sent notifications to applicants. The successful projects that will receive six months 0.5 FTE RSE support are...

- PolyChron: A tool for archaeologists for the management of absolute and relative dating evidence - Bryony Moody (School of Mathematics)
- HYBIRD: a Particle-Fluid Engineering Simulation Tool (HYPFEST) - Alessandro Leonardi (Civil Engineering)
- Tools to analyse splice-aware SLAMseq - Ian Sudbery (School of Biosciences)
```

<!-- List in chronological order -->

- [Register for OpenFest 2024](https://www.sheffield.ac.uk/openresearch/events/openfest-2024-call-proposals)
  - **2024/09/10 - 2024/09/12**. Co-delivered by the University of Sheffield and Sheffield Hallam University, OpenFest is our flagship celebration and exploration of open research, providing an opportunity to explore current issues, share experiences, and consider how open research can be applied in your discipline.

#### Upcoming external events

<!-- List in chronological order -->

- [SSI Fellowship program launch webinar](https://www.software.ac.uk/programmes/fellowship-programme/apply-fellowship-programme)
  - **2024/08/22 @ 11GMT+1** The SSI's Fellowship Programme funds researchers, software engineers and advocates who want to improve computational practice in their domains, fields and/or areas of work.

- [CodeRefinery workshop](https://coderefinery.github.io/2024-09-10-workshop/)
  - **2024/09/10-12 and 17-19**. Free workshop on git, testing, documentation, and more in a workshop - for researchers and scientists.

- [Sustainable HPC State of the Practice](https://sites.google.com/lbl.gov/sustainable-hpc-sop-workshop24/home)

>  - **2024/09/24** The demand for ever more-capable high performance computing (HPC) is driving significant changes across the design and manufacturing space, as manufacturers turn to heterogeneous systems that integrate increasing die-count, from multi-core CPUs to accelerators, traditional memory architectures to high-bandwidth memory, modern interconnects, and massive storage servers. These designs require substantially higher energy, with commensurate methods for managing heat, in increasingly dense packages. To effectively manage and operate these systems, HPC and Data Center (DC) practitioners must balance and coordinate constraints across many domains: environment, utilities, data center, HPC system hardware and software, and end-user applications.

Reviewer: ```suggestion
  - **2024/09/24** The demand for ever more-capable high performance computing (HPC) is driving significant changes across the design and manufacturing space, as manufacturers turn to heterogeneous systems that integrate increasing die-count, from multi-core CPUs to accelerators, traditional memory architectures to high-bandwidth memory, modern interconnects, and massive storage servers. These designs require substantially higher energy, with commensurate methods for managing heat, in increasingly dense packages. To effectively manage and operate these systems, HPC and Data Center (DC) practitioners must balance and coordinate constraints across many domains: environment, utilities, data center, HPC system hardware and software, and end-user applications.
```

Needs 2 spaces not 1 for this to be a child bullet

### Training

For more forthcoming RSE training events, keep an eye on our
[Training](https://rse.shef.ac.uk/training/) page and join our [mailing
list](https://groups.google.com/a/sheffield.ac.uk/g/rse-group) to hear about new events.

<!-- #### External Training -->

<!--#### Research IT Training-->

<!--Research IT courses have adopted a hybrid approach. The team will be providing their courses both online and in -->
<!--person for the first time since March 2020. The team provides a place for beginners or advanced users to expand -->
<!--their knowledge of HPC and different programming languages. The courses are part of the Doctoral Development -->
<!--Programme. For more information please visit our training registration web page (via VPN): -->
<!--[crs.shef.ac.uk](https://crs.shef.ac.uk).-->

<!--If a course is "sold out" please join the wait list by signing up - we regularly email people to encourage those -->
<!--that can no longer attend to cancel. Those on the wait list get early notification when the courses are run -->
<!--again.-->

<!-- ### Opportunities -->

<!-- - [Description](https://www.elsewhere.ac.uk/): -->
<!--   - More Details (optional) -->

> #### Jobs

Reviewer: This is nested below Training currently. 

Either need to uncomment `### Opportunities` above, or make Jobs a h3. 

```suggestion
### Jobs
```
